deliver/lib/deliver/options.rb

```diff
@@ -181,6 +181,11 @@ def self.available_options
                                      description: "Clear all previously uploaded screenshots before uploading the new ones",
                                      is_string: false,
                                      default_value: false),
+        FastlaneCore::ConfigItem.new(key: :skip_unsupported_languages,
```

Comment:
Do we need to have this option? _fastlane_ is going to crash if it needs to activate unsupported languages, so setting this to `false` is just asking for a crash right?